File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1028/BusinessApplication/BA.Web.10.0/Models/Shared/User.shared.cs
namespace $safeprojectname$
{
    /// <summary>
    /// 新增了共用屬性和方法之延伸 User 型別的部分類別。
    /// 這些屬性和方法可同時供伺服器應用程式和用戶端應用程式使用。
    /// </summary>
    public partial class User
    {
        /// <summary>
        /// 傳回使用者顯示名稱 (預設為它的 FriendlyName)，
        /// 如果沒有設定 FriendlyName，則會傳回使用者名稱。
        /// </summary>
        public string DisplayName
        {
            get
            {
                if (!string.IsNullOrEmpty(this.FriendlyName))
                {
                    return this.FriendlyName;
                }
                else
                {
                    return this.Name;
                }
            }
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/3082/BusinessApplication/BA.Web.10.0/Models/Shared/User.shared.vb
Namespace Web

    Partial Public Class User
        ''' <summary>
        ''' Devuelve el nombre para mostrar del usuario, que de forma predeterminada es su FriendlyName.
        ''' Si no se ha establecido FriendlyName, se devuelve el nombre de usuario.
        ''' </summary>
        Public ReadOnly Property DisplayName() As String
            Get
                If Not String.IsNullOrEmpty(Me.FriendlyName) Then
                    Return Me.FriendlyName
                Else
                    Return Me.Name
                End If
            End Get
        End Property
    End Class
End Namespace